torch_xla/csrc/ops/ops.cpp

```diff
@@ -90,6 +90,19 @@ NodePtr TransposeOp(const Value& input) {
                             output_shape, std::move(lower_fn));
 }
 
+NodePtr Sigmoid(const Value& input) {
```

Comment:
Why not reuse `TranslateSigmoid` from `translator.cpp`?

Reply:
Done